integrations/terraform/tfschema/resource153/custom_types.go

```diff
@@ -63,6 +64,7 @@ func CopyToTimestamp(diags diag.Diagnostics, o *timestamppb.Timestamp, t attr.Ty
 	}
 
 	value.Value = (*o).AsTime()
+	value.Format = time.RFC3339
```

Comment:
Nice catch :)

Could you add a test in `testlib` that validates that AccessList import works now?

Reply:
Looks like we don't run enterprise tests https://github.com/gravitational/teleport/pull/44392